src/views/Items/Item.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: (props, propName, componentName) => {
+      if (props.children && props.description) {
```

Comment:
Thinking about our other utils and how we can pull these together.  I think we should actually just this util as a standalone util in the `utils` directory.  We can make an index file later to organize them.  I've added 3 issues for util work that will cover the rest:
#68 CustomPropTypes util
#69 Spreadable props util
#70 Static meta info object